ci-operator/config/openshift-priv/instaslice-operator/openshift-priv-instaslice-operator-next.yaml

```diff
@@ -116,11 +116,15 @@ tests:
       IMAGE_REGISTRY: pipeline
       IMAGE_TAG: e2e-test
     post:
+    - chain: gather-network
```

Comment:
Why do we have a openshift-priv repo for this?

Reply:
Can you revert the changes in openshift-priv?

I think these are auto copied so we shouldn't need to modify these.